snuba/snuba_migrations/outcomes/0006_outcomes_add_size_col.py

```diff
@@ -0,0 +1,69 @@
+from typing import Sequence
+
+from snuba.clickhouse.columns import Column, UInt
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations
+
+
+class Migration(migration.ClickhouseNodeMigration):
+    """
+    Adds back (again) the size, bytes_received columns to match schema in SaaS
```

Comment:
Are `size` and `bytes_received` actually used in prod? Did we decide that it was not worth it to remove these columns from saas outcomes if they are in fact unused?

Reply:
I haven't got a confirmation on that. The plan is to get both to a common state, then afterwards we can decide what to drop.